app/src/main/kotlin/dev/reva/healthexporter/GoogleDriveDestination.kt

```
@@ -0,0 +1,483 @@
+        } catch (e: GoogleDriveException) {
+            DestinationStatus.Unavailable("Google Drive unavailable: ${e.message}", e)
+        } catch (e: Exception) {
+            DestinationStatus.Unavailable("Error verifying Google Drive: ${e.message}", e)
+        }
+    }
+
+    override suspend fun upload(batch: ExportBatch): UploadResult {
+        return try {
+            val startUtc = batch.header.timeWindow.startInclusive.atZone(ZoneOffset.UTC)
+            val year = startUtc.year.toString()
+            val month = String.format("%02d", startUtc.monthValue)
```

detekt (1.23.8)

[warning] String.format("%02d", startUtc.monthValue) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)